libs/elements/src/data-table/vl-data-table.defaults.ts

```diff
@@ -0,0 +1,10 @@
+export const dataTableDefaults = {
+    hover: false,
```

Comment:
Normaal gezien conform andere defaults zou je hier ook `hover: false as boolean,` moeten zetten. Maar dit is eigenlijk out of scope voor dit ticket